src/types/venues.ts

```diff
@@ -46,6 +47,7 @@ export type GenericVenueTemplates = Exclude<
   | VenueTemplate.embeddable
   | VenueTemplate.jazzbar
   | VenueTemplate.partymap
+  | VenueTemplate.animatemap
   | VenueTemplate.posterpage
   | VenueTemplate.themecamp
```

Comment:
These are in alphabetic order:
```suggestion
  | VenueTemplate.animatemap
  | VenueTemplate.embeddable
  | VenueTemplate.jazzbar
  | VenueTemplate.partymap
  | VenueTemplate.posterpage
  | VenueTemplate.themecamp
```